<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonRecipeWidget</class>
 <widget class="QWidget" name="JsonRecipeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Layout_bottom">
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout_head"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>111</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Layout_btns"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>411</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Layout_table"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
